test/unit/features/directives/on.spec.js

```diff
@@ -102,6 +102,24 @@ describe('Directive v-on', () => {
     expect(callOrder.toString()).toBe('1,2')
   })
 
+  it('should support once', () => {
+    const numberPushed = []
+    vm = new Vue({
+      el,
+      template: `
+        <div @click.once="foo">
+        </div>
+      `,
+      methods: {
+        foo () { numberPushed.push(1) }
```

Comment:
you could create a jasmine.createSpy(),  and use `expect(spy).not.toHaveBeenCalled()` to check it.